.github/workflows/on-push-to-release-branch.yaml

```diff
@@ -57,6 +57,17 @@ jobs:
             dotnet pack -c Release -p:Version=${VERSION}
             dotnet nuget push ./bin/Release/Momento.Sdk.${VERSION}.nupkg --source https://api.nuget.org/v3/index.json --api-key=${{secrets.NUGET_API_KEY}}
           popd
+          
+      - name: Pack and Publish for the web SDK
+        run: |
+          set -x
+          pushd src/Momento.Sdk
+            VERSION="${{ needs.release.outputs.version }}"
+            echo "version: ${VERSION}"
+            dotnet build -p:DefineConstants=USE_GRPC_WEB --configuration Release
+            dotnet pack -p:DefineConstants=USE_GRPC_WEB -c Release -p:Version=${VERSION}
+#            dotnet nuget push ./bin/Release/Momento.Sdk.Web.${VERSION}.nupkg --source https://api.nuget.org/v3/index.json --api-key=${{secrets.NUGET_API_KEY}}
```

Comment:
I've executed the steps up to here locally. Do we have a good place to publish this temporarily to test that it works?

Reply:
no, but since it's a net new package, no one will have deps on it yet, so it's not the end of the world if it's broken as long as we fast follow any necessary fixes.